config/locales/user_mailer/en.yml

```diff
@@ -19,20 +19,26 @@ en:
       button: Yes, continue deleting
       cancel_link_text: please cancel
       help_html: If you don’t want to delete your account, %{cancel_account_reset}.
-      intro_html: You are receiving this email because you requested to delete and
-        reset your login.gov account. <br><br> Deleting your account should be your
-        last resort if you are locked out of your account. You will not be able to
-        recover any information linked to your account. Once your account is deleted,
-        you can create a new one using the same email address. <br><br>Are you sure
-        you want to delete your account?
-      subject: Delete your account
+      intro_html: Your 24 hour waiting period has ended. Please complete step 2 of
+        the process.<br><br> If you’ve been unable to locate your authentication methods,
+        select "confirm deletion" to delete your login.gov account.<br><br> In the
+        future, if you need to access participating government websites who use login.gov,
+        you can create a new login.gov account using the same email address after
+        your account is deleted.<br><br>
+      subject: Delete your login.gov account
     account_reset_request:
       cancel: Don't want to delete your account? Sign in to your login.gov account
         to cancel.
       header: Your account will be deleted in 24 hours
```

Comment:
I see the subject is updated, but should the header be as well? It's not really clear from the original content document if "H1" is referring to the subject, the heading, or both. 🤷